docs/eventing/samples/writing-event-source-easy-way/README.md

```diff
@@ -0,0 +1,298 @@
+# Introduction
+
+As stated in [tutorial on writing a Source with a Receive Adapter](../writing-receive-adapter-source/README.md), there are multiple ways to
+create event sources. The way in that tutorial is to create an independent event source that has its own CRD.
+
+In this tutorial though, you will build an event source in Javascript and use it with
+[ContainerSource](../../../eventing/sources/README.md#meta-sources) and / or [SinkBinding](../../../eventing/sources/README.md#meta-sources).
+
+[ContainerSource](../../../eventing/sources/README.md#meta-sources) is an easy way to turn any dispatcher container into an Event Source.
+Similarly, another option is using [SinkBinding](../../../eventing/sources/README.md#meta-sources)
```

Comment:
I thought we were recommending SinkBinding over ContainerSource at this point?

@n3wscott

Reply:
Added a sentence about the preferred way in https://github.com/knative/docs/pull/2512